app/test_main.py

```diff
@@ -1,3 +1,53 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+def test_get_human_age_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0], \
+        "Expected [0, 0] for 0 cat and 0 dog age"
```

Comment:
suggest you use braces
```suggestion
    assert (get_human_age(0, 0) == [0, 0], 
        "Expected [0, 0] for 0 cat and 0 dog age)"
```